services/libs/aster-std/src/sched/nice.rs

```diff
@@ -0,0 +1,101 @@
+// SPDX-License-Identifier: MPL-2.0
+
+use crate::prelude::*;
+
+/// The process scheduling priority value.
+///
+/// It is a value in the range -20 to 19, with -20 being the highest priority
+/// and 19 being the lowest priority. The smaller values give a process a higher
+/// scheduling priority.
+#[derive(Copy, Clone, Debug, PartialEq, Eq, PartialOrd, Ord)]
+pub struct Nice {
+    raw: i32,
+}
+
+impl Nice {
+    const MIN: i32 = -20;
+
+    const MAX: i32 = 19;
+
+    /// Returns the highest priority.
+    pub fn highest_priority() -> Self {
```

Comment:
I think having the `highest_priority` and `lowest_priority` methods are redundant with the `MAX` and `MIN`. From the user side, typing `Nice::MAX` is faster than `Nice::highest_priority()`. The same argument can be made to `RlimitNice`.

```rust
impl Nice {
    /// The maximum nice, whose value is 19.
    pub const MAX: Self = Nice { raw: 19 };
}
```

Reply:
Done